performance-storage-service/pss_project/api/models/rest/BinaryMetricsRest.py

```diff
@@ -0,0 +1,10 @@
+from pss_project.api.models.rest.BaseRest import BaseRest
+
+
+class BinaryMetricsRest(BaseRest):
+    def __init__(self, metadata, timestamp, metrics):
+        super().__init__(metadata, timestamp)
+        self.metrics = metrics
+
+    def convert_metrics_to_dict(self, metrics):
+        return metrics
```

Comment:
I am a bit confused what is the purpose of this function?

Reply:
I am just creating default behavior for the function because it gets called in `convert_db_to_json`. Some subclasses will override this method like here https://github.com/cmu-db/noisepage-test/blob/d1e5788ae343246f99b84597afe80f2ade84f8bf/performance-storage-service/pss_project/api/models/rest/OLTPBenchRest.py#L29 but others wont need to override the method (like BinaryMetrics)